### ISSUE TITLE
Performance in isic-skin-cancer/analyze_isic.ipynb is too different from the one in the paper

### ISSUE THREAD

**Issue by Juna2** [laura-rieger/deep-explanation-penalization#9]
Hi, long time no see
Could you tell me in more detail that how you get the result of isic datset in the paper?
For example, hyperparameters like batch_size, learning_rate, reg_rate extra.
Cause the performance in the paper and the ipynb file is quite different.
Thanks in advance!

**Comment by laura-rieger** [comment]
Hi Jeon,
Unfortunately this does seem to have been caused by an error in the evaluation (during refactoring the order in which classes are concatenated was flipped, resulting in training images in the test set). 
Luckily this only affects the absolute metrics while the ranking and differences of CDEP and the baselines stay consistent. 
Running the script as it is will get you the results in the updated notebook.
We are in the process of updating the paper to reflect those changes.   
Thank you for alerting us to this issue!!
Best, Laura

**Comment by Juna2** [comment]
Umm so do you mean that the result in the paper came from the dataset which train data has flowed into its testset?

**Comment by laura-rieger** [comment]
The results in the notebook are correct. The results in the paper as of right now are incorrect (higher for all methods, as there are training samples in the test set).

**Comment by Juna2** [comment]
Thank you for the information!!!
